app/src/main/java/com/connorlay/knitgrid/activities/PatternGridViewActivity.java

```diff
@@ -21,5 +26,17 @@ protected void onCreate(Bundle savedInstanceState) {
         transaction.replace(R.id.activity_pattern_grid_view_content_frame_layout, new PatternGridFragment());
         transaction.addToBackStack(null);
         transaction.commit();
+
+        if (isFirstLaunch()) {
```

Comment:
Yeah that's a good point. We should ask Nick where app initialization usually lives.